src/settings/MARCFieldProtection/MARCFieldProtection.js

```diff
@@ -39,12 +36,16 @@ const validateSubfield = (subfieldValue, fieldValue) => {
   let pattern = /^[*\w]$/;
   let errorMessage = <FormattedMessage id="ui-data-import.validation.enterAsteriskOrAlphanumeric" />;
 
-  if (fieldValue === '*') {
+  if (!subfieldValue) {
+    errorMessage = <FormattedMessage id="stripes-core.label.missingRequiredField" />;
+  }
+
+  if (subfieldValue && fieldValue === '*') {
```

Comment:
```suggestion
  else if (fieldValue === '*') {
```
I think here the code can be simplified as in case the `subfieldValue` is falsy the condition above will take place.